src/app/shared/entry/GA4GHFiles.state.service.ts

```diff
@@ -63,8 +63,12 @@ export class GA4GHFilesStateService {
    * @memberof GA4GHFilesStateService
    */
   public injectAuthorizationToken(ga4ghService: GA4GHService) {
-    ga4ghService.defaultHeaders = ga4ghService.defaultHeaders.set('Authorization',
-      ga4ghService.configuration.apiKeys['Authorization']);
+    const auth = ga4ghService.configuration.apiKeys['Authorization'];
+    if (auth) {
```

Comment:
Could simplify to one liner like the change in my-entry

Reply:
As an opinion, I think this might be getting a bit long/unreadable for a ternary at, roughly 100 characters.